recommender/beta/cloud-client/src/main/java/com/example/recommender/ListRecommendations.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.recommender;
+
+// [START recommender_list_recommendations]
+import com.google.cloud.recommender.v1beta1.ListRecommendationsRequest;
+import com.google.cloud.recommender.v1beta1.Recommendation;
+import com.google.cloud.recommender.v1beta1.RecommenderClient;
+import java.io.IOException;
+
+public class ListRecommendations {
+
+  // List recommendations for a specified project, location, and recommender
+  public static void listRecommendations(String projectId, String location, String recommender) {
+    try (RecommenderClient recommenderClient = RecommenderClient.create()) {
```

Comment:
Please add a comment explaining why this client is used in a try-catch - see [Client Initialization](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#client-initialization)

Reply:
Since main reason is not setting up authentication for client init failure, I'm just letting the error bubble up.